app/views/state_file/questions/id_review/edit.html.erb

```diff
@@ -2,6 +2,20 @@
 <% content_for :card do %>
   <%= render "state_file/questions/shared/review_header" %>
 
+  <% if Flipper.enabled?(:show_retirement_ui) && current_intake.calculator.line_or_zero(:ID39R_B_LINE_8e).positive? %>
+    <div id="qualified-retirement-benefits-deduction" class="white-group">
+      <div class="spacing-below-5">
+        <p class="text--bold spacing-below-5"><%= t(".qualified_retirement_benefits_deduction") %></p>
+        <p class="text spacing-below-2"><%= t(".qualified_retirement_benefits_deduction_explain") %></p>
+
+        <p class="text--bold spacing-below-5"><%= t(".qualified_disabled_retirement_benefits") %></p>
+        <p><%= number_to_currency(current_intake.calculator.line_or_zero(:ID39R_B_LINE_8f), precision: 2) %></p>
+        <% id_disability_or_retirement_link = StateFile::Questions::IdDisabilityController.meets_age_requirements?(current_intake) ?  StateFile::Questions::IdDisabilityController.to_path_helper(return_to_review: "y") : StateFile::Questions::IdRetirementAndPensionIncomeController.to_path_helper(return_to_review: "y", index: 0) %>
```

Comment:
if they see this card, they initially entered the IdDisability/IdRetirementAndPensionIncomeController.
The logic to return to the edit screen is much simpler, we just ask if they are between 62-65 (`meets_age_requirements?`), and if so - go to disability controller, or go straight to IdRetirementAndPensionIncomeController (for over 65+)